tests/main/econnreset/task.yaml

```diff
@@ -22,24 +25,25 @@ execute: |
 
     echo "Wait until the download started and downloaded more than 1 MB"
     echo "starting: $(date)"
-    for _ in $(seq 400); do
+    for _ in $(seq 120); do
         partial=$(find . -name 'test-snapd-huge_*.snap.partial' | head -1)
         if [ -n "$partial" ] && [ "$(stat -c%s "$partial")" -gt "$(( 1024 * 1024 ))" ]; then
             break
         fi
-        sleep 0.1
+        sleep 0.5
     done
 
     if [ ! -f "$partial" ] || [ "$(stat -c%s "$partial")" -eq 0 ]; then
         echo "Partial file $partial did not start downloading, test broken"
-        kill -9 "$(pidof snap)"
+        kill -9 "$(pgrep -f 'snap download')" || true
```

Comment:
This is redundant, no? I mean, restore will also kill it? And we exit right away?